README.md

```diff
@@ -31,10 +32,11 @@ import (
 )
 
 func main() {
-    // Create MCP server
+    // Create a new MCP server
     s := server.NewMCPServer(
         "Demo 🚀",
         "1.0.0",
+        server.WithToolCapabilities(false),
```

Comment:
> Servers that support tools MUST declare the tools capability

See: https://modelcontextprotocol.io/specification/2024-11-05/server/tools#capabilities

It is good if the examples follow the spec correctly.